src/assets/localization/en.json

```diff
@@ -361,7 +362,7 @@
   "page.profile.menu.applications.actions.deactivate": "Deactivate Application",
   "page.profile.menu.applications.actions.delete": "Delete Application",
   "page.profile.menu.applications.actions.edit": "Edit Form",
-  "page.profile.menu.applications.actions.header": "Application for work:",
+  "page.profile.menu.applications.actions.header": "{{title}}:",
```

Comment:
No need of this anymore, it will be just BE response

Reply:
yep, removed it!